ansible/vars/enikshay/enikshay_public.yml

```diff
@@ -77,9 +77,10 @@ keystore_file: '../config/DimagiKeyStore'
 encrypted_root: '/opt/data'
 
 backup_blobdb: False
-backup_postgres: pgbackrest
+backup_postgres: plain
 postgresql_backup_days: 2
 postgresql_backup_weeks: 2
+postgresql_backup_master: True
```

Comment:
why are we backing up directly from the master on enikshay

Reply:
i first decided to do this in order to avoid missing wal issues we got in icds. can the slave keep them as well? i can test to make sure we don't *have* to use master.

Reply:
Correct me if I'm wrong but since we're using `-Xs` as an option for pg_basebackup PostgreSQL will keep as many segments as are required for the backup so we shouldn't need to have a high value for `wal_keep_segments`

Reply:
-Xs doens't allow compression, also before 9.6 doesn't guarantee keeping all necessary wals.  WIthout compression we couldn't even afford one backup.  So we use -Xf

Reply:
I think the standby can keep them as well. I guess its worth testing but we do it now from the standby so i don't see why they couldnt. And if it could, then we would be able to back up locally and not worry about load on the master.

Reply:
yeah, only we don't have much space in the standby, but neither do we in the master so... ok, done.